caesar_cipher.rb

```diff
@@ -0,0 +1,97 @@
+# Program that will cipher a String that the user will pass to it.
+
+class Caesar
+    ALPHABET_UPPERCASE = Array ('A'..'Z')
+    ALPHABET_LOWERCASE = Array ('a'..'z')
+
+    def initialize(text, num)
+        @text = text
+        @num = num
+    end
+
+    # Method that ciphers the text that we give it
+    def cipher
+        # It will check if there is text to cipher and if there is a valid number to change the letters
+        return puts "The text given is null and/or the number given is 0." if text == "" || num.zero?
+        text_split = text.split('')
+        puts refactor_text(text_split)
+    end
+
+    private
+
+    attr_reader :text, :num
+    
+    # Will create a hash for the uppercase letter or the lower case ones
+    def fill_alphabet_upper
+        return @fill_alphabet_upper if defined? @fill_alphabet_upper
+        @fill_alphabet_upper = Hash.new
```

Comment:
You can just use  `{}`
 ```suggestion
        @fill_alphabet_upper = {}
```

Reply:
+1